packages/fxa-content-server/app/scripts/views/pair/pairing-totp-mixin.js

```diff
@@ -6,6 +6,9 @@ import AuthErrors from '../../lib/auth-errors';
 
 export default function () {
   return {
+    hasTotpEnabledOnAccount(profile) {
+      return profile.authenticatorAssuranceLevel > 1;
```

Comment:
`authenticatorAssuranceLevel` should give you the current verification level of the sessionToken. I think you need to check that `authenticationMethods` contains `otp` which would mean that the user has enabled TOTP on the account.